src/Pages/FeedPage/FeedPageProfile.jsx

```diff
@@ -0,0 +1,35 @@
+import useRequest from "../../hooks/useRequest";
+import { useEffect } from "react";
+function FeedPageProfile() {
+  const {
+    data: feedProfileData,
+    error,
+    isLoading,
+    request,
+  } = useRequest({
+    method: "GET",
+    url: "subjects/3825",
+  });
+
+  useEffect(() => {
+    request();
+  }, []);
+
+  useEffect(() => {
+    console.log(error);
+  }, [error]);
+
+  useEffect(() => {
+    console.log("isLoading : " + isLoading);
+  }, [isLoading]);
+  return (
+    <div>
+      <div>
+        <img src={feedProfileData?.imageSource} />
+        <p>{feedProfileData?.name}</p>
```

Comment:
```suggestion
        {feedProfileData?.imageSource && <img src={feedProfileData?.imageSource} />}
        <p>{feedProfileData?.name ?? error}</p>
```
요렇게 처리하는게 쪼끔 더 안정성이 있지 않을까 조심스럽게 제안드려봅니다!
(사실 알못입니다)

Reply:
(저도 알못입니다)
저도 아름님 말씀처럼 && 연산자 이용하는게 더 안정적일거 같다고 생각합니다.

나중에 시간 되면 분기처리 적용을 고려해보시는 것도 좋을 것 같습니다.